packages/core/src/internal/base/css-gap.base.spec.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { LitElement, html } from 'lit-element';
+import { createTestElement, removeTestElement, componentIsStable } from '@cds/core/test/utils';
+import { applyCSSGapShim } from './css-gap.base.js';
+import { browserFeatures } from '../utils/supports.js';
+
+class DemoComponent extends LitElement {}
+
+// register the element just for testing purposes
+window.customElements.define('cds-css-gap-shim', applyCSSGapShim(DemoComponent));
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'cds-css-gap-shim': DemoComponent;
+  }
+}
+
+describe('CSS Flex Gap Shim', () => {
+  let testElement: HTMLElement;
+  let component: DemoComponent;
+
+  beforeEach(async () => {
+    testElement = await createTestElement(html`<cds-css-gap-shim></cds-css-gap-shim>`);
+    component = testElement.querySelector<DemoComponent>('cds-css-gap-shim');
+  });
+
+  afterEach(() => {
+    removeTestElement(testElement);
+  });
+
+  it('should not apply css gap shim attr if not supported', async () => {
+    await componentIsStable(component);
+    expect(component.getAttribute('_nfg')).toBe(null);
+  });
+
+  it('should apply css gap shim attr if not supported', async () => {
+    (browserFeatures as any).supports.flexGap = false; // force false flex gap support
+    await componentIsStable(component);
+    expect(component.getAttribute('_nfg')).toBe(null);
```

Comment:
Is this test case identical to the one above? Shouldn't the `_nfg` attribute have empty value (`''`) instead of `null` if it does not support flex gap?